Description: use https at some points
 - all links from links.h work fine with https
 - sig.cpp work fine with https afaict
 - youtube: set use_https_* to true by default
 

Author: Alf Gaida <agaida@siduction.org>

Last-Update: 2018-11-10

--- a/src/links.h
+++ b/src/links.h
@@ -19,8 +19,8 @@
 #ifndef LINKS_H
 #define LINKS_H
 
-#define URL_HOMEPAGE "http://www.smplayer.info"
-#define URL_FORUM "http://forum.smplayer.info"
+#define URL_HOMEPAGE "https://www.smplayer.info"
+#define URL_FORUM "https://forum.smplayer.info"
 #define URL_TRANSLATORS URL_HOMEPAGE"/translators.php"
 #define URL_FIRST_STEPS URL_HOMEPAGE"/first-steps.php"
 #define URL_FAQ URL_HOMEPAGE"/faq.php"
@@ -28,8 +28,8 @@
 #define URL_THANK_YOU URL_HOMEPAGE"/thank-you.php"
 #define URL_OPENSSL_INFO URL_HOMEPAGE"/openssl.php"
 #define URL_DONATE URL_HOMEPAGE"/donate.php"
-#define URL_CHROMECAST "http://chromecast.smplayer.info"
-#define URL_YT_CODE "http://updates.smplayer.info/yt.js"
-#define URL_VERSION_INFO "http://updates.smplayer.info/version_info.ini"
+#define URL_CHROMECAST "https://chromecast.smplayer.info"
+#define URL_YT_CODE "https://updates.smplayer.info/yt.js"
+#define URL_VERSION_INFO "https://updates.smplayer.info/version_info.ini"
 
 #endif
--- a/src/youtube/retrieveyoutubeurl.cpp
+++ b/src/youtube/retrieveyoutubeurl.cpp
@@ -134,7 +134,7 @@
 		video_url.startsWith("youtu.be") || video_url.startsWith("www.youtu.be") || 
 		video_url.startsWith("y2u.be") || video_url.startsWith("www.y2u.be"))
 	{
-		video_url = "http://" + video_url;
+		video_url = "https://" + video_url;
 	}
 
 	//qDebug() << "RetrieveYoutubeUrl::getVideoID: fixed url:" << video_url;
 
